src/Css.elm

```diff
@@ -635,6 +636,11 @@ Multiple CSS properties use these values.
 @docs whiteSpace
 @docs pre, preWrap, preLine, nowrap
 
+
+## Float
+
+@docs float
```

Comment:
@rtfeldman do I need to add a line containing the values `left_`, `right_`, and `none` here? Since those are already defined in with other CSS properties

Reply:
Nah, I'd say leave them where they are! 👍